src/mugs.js

```diff
@@ -473,6 +473,15 @@ define([
             }
             return this.parentMug && this.parentMug.isInRepeat();
         },
+        /**
+         * Check if mug is referenced by other mugs
+         *
+         * @param except - Array of mugs to exclude when looking for references.
+         * @returns Boolean
+         */
+        isReferencedByOtherMugs: function (except) {
```

Comment:
really nitpicky: in a perfect world should mugs know what a form is like this? I usually think of it as forms encapsulate mug and mugs shouldn't necessarily know about others. I'm sure that convention is broken all over though or I may not be thinking of other use cases